### DISCUSSION TITLE
How about adding checkbox in index.html to disable entering binding mode by 3 times-power-on?

### DISCUSSION THREAD

**Discussion by Ncerzzk** [ExpressLRS/ExpressLRS#2782] [General]
In my DIY RX device, I found it's easy to get into the binding mode, and the config will be cleared.
The cause may be the glitch when power on? 

Are there other one met this issue? and how about adding a switch to shut it down?

**Comment by pkendall64** [comment]
This is a known issue and there is a fix in the 3.x.x-maintenance branch, which should be released fairly soon as 3.4.3

**Reply by Ncerzzk** [reply]
Wow, thank you, I found this commit, I will try it.
